src/cart/utils/memcheck-cart.supp

```diff
@@ -594,3 +594,33 @@
    fun:__tsan_go_atomic64_compare_exchange
    fun:racecall
 }
+{
+   go 1.22.3 race
+   Memcheck:Value8
+   fun:__tsan_write_pc
+   fun:racecall
+}
+{
+   go 1.22.3 race
+   Memcheck:Value8
+   fun:_ZN6__tsan24TraceRestartMemoryAccessEPNS_11ThreadStateEmmmm
+   fun:racecall
+}
+{
+   go 1.22.3 race
+   Memcheck:Value8
+   fun:_ZN6__tsan18MemoryAccessRangeTILb0EEEvPNS_11ThreadStateEmmm
+   fun:racecall
+}
+{
+   go 1.22.3 race
+   Memcheck:Value8
+   fun:_ZN6__tsan24TraceRestartMemoryAccessEPNS_11ThreadStateEmmmm
```

Comment:
Is this the same as the second one above? If so the duplicate could be removed.

Reply:
Removed duplicate.